core/src/main/scala/org/apache/spark/sql/TiStrategy.scala

```diff
@@ -98,6 +99,10 @@ case class TiStrategy(getOrCreateTiContext: SparkSession => TiContext)(sparkSess
   private def useStreamingProcess(): Boolean =
     sqlConf.getConfString(TiConfigConst.COPROCESS_STREAMING, "false").toLowerCase.toBoolean
 
+  private def isEnableChunk(): Boolean =
+    sqlConf.getConfString(TiConfigConst.ENABLE_CHUNK, "false").toLowerCase.toBoolean ||
```

Comment:
This `||` is confusing. Consider `ENABLE_CHUNK` is `true` in config file, and user sets it to `false` through shell or client, expecting his latter set would override the one in config file. What will he see?

Reply:
+1 for confusing